src/SelfInitializingFakes/SelfInitializingFake.of.T.cs

```diff
@@ -125,142 +95,25 @@ public void Dispose()
         {
             if (this.IsRecording)
             {
-                if (this.recordingException != null)
+                if (this.recordingRule.RecordingException != null)
                 {
                     throw new PlaybackException(

```

Comment:
I realize it's not the subject of this PR, but shouldn't it be a `RecordingException`? At least that's what I'd expect.

Reply:
Yes! How embarrassing! Thanks. #48.

Reply:
Or not.